src/main/java/org/jboss/pnc/bifrost/Config.java

```diff
@@ -14,14 +14,16 @@
 @ApplicationScoped
 public class Config {
 
-    @ConfigProperty(name = "bifrost.defaultSourceFetchSize", defaultValue = "100")
+    @ConfigProperty(name = "bifrost.defaultSourceFetchSize",
+                    defaultValue = "100")
     int defaultSourceFetchSize;
 
-    @ConfigProperty(name = "bifrost.maxSourceFetchSize", defaultValue = "1000")
+    @ConfigProperty(name = "bifrost.maxSourceFetchSize",
+                    defaultValue = "1000")
     int maxSourceFetchSize;
 
-    @ConfigProperty(name = "bifrost.sourcePollThreads", defaultValue = "4")
+    @ConfigProperty(name = "bifrost.sourcePollThreads",
```

Comment:
This one is not long enough for wrapping ?

Reply:
I will look into it again, but I think eclipse doesn't provide such a setting

Reply:
Given the alignment you want for methods and functions, we can do the same thing for annotations, wrapping if necesasry